Modules/posixmodule.c

```diff
@@ -7946,7 +7946,7 @@ os_getgid_impl(PyObject *module)
 #endif /* HAVE_GETGID */
 
 
-#ifdef HAVE_GETPID
+#if defined(HAVE_GETPID) || defined(MS_WINDOWS_UWP)
```

Comment:
As mentioned in the pyconfig.h comment, let's keep `HAVE_GETPID` defined and only use the new variable close to the actual API usage.